.github/workflows/buildAndPush.yaml

```diff
@@ -19,10 +19,15 @@ jobs:
         uses: borales/actions-yarn@v2.0.0
         with:
           cmd: lint
+      - name: Get release version
+        id: get_version
+        run: echo ::set-env name=REACT_APP_VERSION::$(git describe --abbrev=0)
```

Comment:
This effectively takes latest `git tag`, I think more relevant is referencing via git's `sha`